uber/site_sections/registration.py

```diff
@@ -719,7 +719,7 @@ def shifts(self, session, id, shift_id='', message=''):
                                            .outerjoin(Job.shifts)
                                            .filter(Job.start_time > localized_now() - timedelta(hours=2),
                                                    Job.location.in_(attendee.assigned_depts_ints),
-                                                   *([] if attendee.trusted else [Job.restricted == False]))
+                                                   *([] if Job.location.in_(attendee.trusted_depts) else [Job.restricted == False]))
                                            .group_by(Job.id)
```

Comment:
I'm not sure either, since ``location`` is an integer field, and ``trusted_depts`` is a textual field.  I don't know enough about Postgres to know whether it's okay with that.  If not then I'm sure it's a simple fix, there's gotta be something you can call via ``sqlalchemy.func`` to make it work if this gives an error as-is.